src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs

```diff
@@ -756,5 +761,18 @@ private HubMessage ApplyHeaders(HubMessage message, Dictionary<string, string> h
 
             return message;
         }
+
+        internal static JsonSerializerOptions CreateDefaultSerializerSettings()
+        {
+            var options = new JsonSerializerOptions();
+            options.WriteIndented = false;
+            options.ReadCommentHandling = JsonCommentHandling.Disallow;
+            options.AllowTrailingCommas = false;
+            options.IgnoreNullValues = false;
+            options.IgnoreReadOnlyProperties = false;
+            // TODO: camelCase
```

Comment:
Tracking this somewhere?

Reply:
Yes, in my head. I'll be doing this today so no need to track it.